SUMMARY.md

```diff
@@ -1,6 +1,6 @@
 # Table of contents
 
-* [Introduction](README.md)
+* [Introduction](README.markdown)
```

Comment:
@AustinBwanika This file should not be changed

Reply:
@ssmusoke I have reverted what I did and it's back to it's original format.